x/bank/keeper/genesis.go

```diff
@@ -42,10 +43,15 @@ func (k BaseKeeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 
 // ExportGenesis returns the bank module's genesis state.
 func (k BaseKeeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
+	totalSupply, _, err := k.GetPaginatedTotalSupply(ctx, &query.PageRequest{})
```

Comment:
I think this has a problem, when there is no limit set the Limit is set to Default which is 100. This would mean that when you export a chain that it has more than 100 they will never end in the genesis.

@aaronc @amaurym correct me if I am wrong.

Reply:
yeah you're right, I'll iterate across the result and append

Reply:
What about using Limit = max_uint64? That would iteratate over all supplies, right?